source/Mission.cpp

```diff
@@ -1378,6 +1383,11 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 		subs["<waypoints>"] = systems;
 	}
 
+	// Done making subs, so expand the phrases and recursively substitute.
+	for(auto &keyValue : subs)
```

Comment:
```suggestion
	for(const auto &keyValue : subs)
```
same here.

or am I overlooking something?